### PR TITLE
Fix a failing functional test that looked for the count of services

### PR DESCRIPTION
**Pull request by jonodrew**
Laurence and I did some digging and tweaked the 'User is able to click on several random filters' feature to look for invisible ids. This seems to have fixed a previously flaky test

### CODE REVIEW

File: features/step_definitions/common_steps.rb [hunk 1]
```diff
@@ -506,8 +506,9 @@
     page.find_by_id(
       page.all(
         :xpath,
-        "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']").sample["aria-controls"]
-    ).text
+        "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']").sample["aria-controls"],
+        visible: :true
```
Comment (lfdebrux) [inline features/step_definitions/common_steps.rb]: `visible: true` is the default, I think we wanted this `visible: :all` is what we wanted for Chrome

```suggestion
        visible: :all
```

**Comment by lfdebrux** [comment]
I've tested this locally and it works

File: features/step_definitions/common_steps.rb [hunk 2]
```diff
@@ -506,8 +506,9 @@
     page.find_by_id(
       page.all(
```
Comment (risicle) [inline features/step_definitions/common_steps.rb]: Where's the closing bracket corresponding to this?

Comment (jonodrew) [reply features/step_definitions/common_steps.rb]:
```
page.find_by_id(
  page.all(
    :xpath,
    "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']"
  ).sample["aria-controls"], visible: :all
).text(:all)
```
tl;dr in the middle of line 509

Comment (risicle) [reply features/step_definitions/common_steps.rb]: Ahhhhhhhhhhh

**Comment by benvand** [comment]
I'm happy to take this out while @jonodrew is away.

LGTM
